spec/clockwork/test/manager_spec.rb

```diff
@@ -23,6 +24,24 @@
 
       its(:end_time) { should eq end_time }
     end
+
+    context "start_time provided, while end_time is not" do
+      let(:opts) { { start_time: 5.minutes.ago } }
+
+      before { Timecop.freeze }
+      after { Timecop.return }
+
+      its(:start_time) { should eq opts[:start_time] }
+      its(:end_time) { should eq Time.current }
+    end
+
+    context "start_time happens after the end_time" do
+      let(:opts) { { start_time: 5.minutes.ago, end_time: 10.minutes.ago } }
+
+      it "raises with useful message" do
+        expect { subject }.to raise_error("End time can't precede start time.")
```

Comment:
Minor point, but I'd prefer to use the named subject (`manager`) than the implicit `subject`.

Reply:
fixed